packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanBar.test.js

```diff
@@ -73,20 +73,18 @@ describe('<SpanBar>', () => {
   };
 
   it('renders without exploding', () => {
-    const wrapper = mount(<SpanBar {...props} />);
-    expect(wrapper).toBeDefined();
-    const { onMouseOver, onMouseOut } = wrapper.find('.SpanBar--wrapper').props();
-    const labelElm = wrapper.find('.SpanBar--label');
-    expect(labelElm.text()).toBe(shortLabel);
-    onMouseOver();
-    expect(labelElm.text()).toBe(longLabel);
-    onMouseOut();
-    expect(labelElm.text()).toBe(shortLabel);
+    render(<SpanBar {...props} />);
+    const labelElm = screen.getByText(shortLabel);
+    expect(screen.getByText(shortLabel)).toBeInTheDocument();
```

Comment:
```suggestion
    expect(labelElm).toBeInTheDocument();
```

Reply:
Here, we are checking if shortLabel is present before mouse in, then checking for longLabel on mouse in, then again for shortLabel on mouse out, to verify shortLabel and longLabel toggling.

Reply:
oh, got it.